onnxruntime/contrib_ops/cuda/bert/attention.cc

```diff
@@ -64,6 +64,10 @@ Status Attention<T>::ComputeInternal(OpKernelContext* context) const {
   int past_sequence_length = 0;
   Tensor* present = GetPresent(context, past, batch_size, head_size, sequence_length, past_sequence_length);
 
+  if (input->SizeInBytes() == 0) {
```

Comment:
>input->SizeInBytes() [](http://example.com/codeflow?start=6&length=20)

shape.Size() is a better choice. Also make sure the output even empty is created